docs/topics/forms/modelforms.txt

```diff
@@ -788,6 +788,12 @@ with the ``Author`` model. It works just like a regular formset::
     means that a model formset is an extension of a basic formset that knows
     how to interact with a particular model.
 
+.. note::
+
+    When using :ref:`multi-table inheritance <multi-table-inheritance>`, forms
+    generated by a formset factory will contain a parent link field (by default
+    ``<parent_class_name>_ptr``) instead of an ``id`` field.
```

Comment:
`model_name` implies that it's lower cased.

https://github.com/django/django/blob/b0e92fb49d1c6f5c18e36f91c18d3ed981b47ae9/django/db/models/options.py#L157

```suggestion
    ``<parent_model_name>_ptr``) instead of an ``id`` field.
```

Reply:
Agreed, I updated PR.